22.09.2025/merge_sort.c

```diff
@@ -0,0 +1,57 @@
+#include "sorter.h"
+#include <stdio.h>
+#include <string.h>
+
+#define BUFFER_CAPACITY (100)
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
+
+/*
+ * Reads integers until newline in array 'arr'
+ * of size 'capacity', returns the number of integers
+ */
+int scanNumbers(int* arr, int capacity)
+{
+    char buf[BUFSIZ];
+    printf("Enter numbers:\n");
+    fgets(buf, sizeof(buf), stdin);
+
+    int num = 0;
+    int offset = 0;
+    int number_offset = 0;
```

Comment:
```suggestion
    int numberOffset = 0;
```
И далее по тексту поменять весь snake_case на camelCase